src/routes/view-content/[guideId]/[bibleSection]/guide-content/UwTranslationContent.svelte

```diff
@@ -92,6 +111,29 @@
         }
     }
 
+    function getUsedInVerses(
+        verseInfo: ResourceContentInfoWithFrontendData['verses'],
+        currentBibleSection: BibleSection | null,
+        bookCodesToNames: Map<string, string> | undefined
+    ) {
+        var usedInVerses = `<b>${$translate('page.passage.guide.usedInVerses.value')}</b><div class="ml-4 mt-2 mb-4">`;
+        if (currentBibleSection?.bookCode) {
+            var bibleBook = bookCodesToNames?.get(currentBibleSection?.bookCode);
+            verseInfo.forEach((ref, i) => {
+                if (i === 0) {
+                    usedInVerses += bibleBook + ' ';
+                }
+                usedInVerses += ref.chapter + ':' + ref.verse;
```

Comment:
Should've caught this in my last review but we should be using `handleRtlVerseReferences` from here: https://github.com/BiblioNexusStudio/well-web/blob/master//Users/kylegrinstead/Developer/well-web/src/lib/utils/language-utils.ts#L28

That way it'll work in RTL languages with the verse reference colon there.